packages/sdk/shell/src/panels/IdentityPanel/steps/IdentityActionChooser.tsx

```diff
@@ -44,83 +38,28 @@ export const IdentityActionChooser = (props: IdentityPanelStepProps) => {
+  onCreateInvitationClick,
+  ...deviceListProps
 }: IdentityActionChooserImplProps) => {
-  const { t } = useTranslation('os');
-
-  // const doneAction = (
-  //   <PanelAction aria-label={t('done label')} onClick={onDone} disabled={!active} data-testid='identity-panel-done'>
-  //     <Check weight='light' className={getSize(6)} />
-  //   </PanelAction>
-  // );
   return (
-    <div role='none' className='grow flex flex-col gap-1'>
-      <DensityProvider density='coarse'>
-        <div className='grow justify-center flex flex-col gap-1'>
-          <Action data-testid='manage-profile' onClick={() => send?.({ type: 'chooseProfile' })} classNames='plb-4'>
-            <UserGear className={getSize(6)} />
-            <span className='grow mli-3'>{t('choose profile label')}</span>
-            <CaretRight weight='bold' className={getSize(4)} />
-          </Action>
-          {agentHostingEnabled && (
-            <Action data-testid='manage-agent' onClick={() => send?.({ type: 'chooseAgent' })} classNames='plb-4'>
-              <HardDrive className={getSize(6)} />
-              <span className='grow mli-3'>Manage Agent</span>
-              <CaretRight weight='bold' className={getSize(4)} />
-            </Action>
-          )}
-          <Action
-            disabled={!active}
-            data-testid='devices-panel.create-invitation'
-            onClick={onCreateInvitationClick}
-            classNames='plb-4'
-          >
-            <Plus className={getSize(6)} />
-            <span className='grow mli-3'>{t('choose add device label')}</span>
-            <CaretRight weight='bold' className={getSize(4)} />
-          </Action>
-          <Action
-            data-testid='devices-panel.join-new-identity'
-            onClick={() => send?.({ type: 'chooseJoinNewIdentity' })}
-            classNames='plb-4'
-          >
-            <Intersect className={getSize(6)} />
-            <span className='grow mli-3'>{t('choose join new identity label')}</span>
-            <CaretRight weight='bold' className={getSize(4)} />
-          </Action>
-          <Action data-testid='manage-devices' onClick={() => send?.({ type: 'chooseDevices' })} classNames='plb-4'>
-            <Devices className={getSize(6)} />
-            <span className='grow mli-3'>{t('choose devices label')}</span>
-            <CaretRight weight='bold' className={getSize(4)} />
-          </Action>
-          <Action
-            data-testid='devices-panel.sign-out'
-            onClick={() => send?.({ type: 'chooseResetStorage' })}
-            classNames='plb-4'
-          >
-            <Power className={getSize(6)} />
-            <span className='grow mli-3'>{t('choose sign out label')}</span>
-            <CaretRight weight='bold' className={getSize(4)} />
-          </Action>
-        </div>
-      </DensityProvider>
-      {/* <PanelActions>{doneActionParent ? cloneElement(doneActionParent, {}, doneAction) : doneAction}</PanelActions> */}
-    </div>
+    <DeviceList
```

Comment:
its a little weird to me that the `IdentityActionChooser` is actually just the device list, could we avoid the wrapping and just directly render the device list? if not maybe rename the wrapper?

Reply:
Based on your comment about the agent, this will host that as well. Maybe ‘action chooser’ has become a misnomer, since this is really more like “identity landing page”.